test/compress/issue-208.js

```diff
@@ -27,3 +27,28 @@ do_update_rhs: {
         MY_DEBUG += 0;
     }
 }
+
+mixed: {
+    options = {
+        global_defs: { DEBUG: 0 }
+    }
+    input: {
```

Comment:
Please add this to the beginning of input:
```
const DEBUG = true;
```
It should also generate a warning.

Could you also add `OTHER: 1` to  global_defs and the following line to the test:
```
var OTHER;
```
To ensure it also produces a warning.